tls/s2n_crl.c

```diff
@@ -91,6 +91,39 @@ int s2n_crl_get_issuer_hash(struct s2n_crl *crl, uint64_t *hash)
     return S2N_SUCCESS;
 }
 
+int s2n_crl_validate_active(struct s2n_crl *crl)
+{
+    POSIX_ENSURE_REF(crl);
+    POSIX_ENSURE_REF(crl->crl);
+
+    ASN1_TIME *this_update = X509_CRL_get_lastUpdate(crl->crl);
+    POSIX_ENSURE_REF(this_update);
+
+    int ret = X509_cmp_time(this_update, NULL);
+    POSIX_ENSURE(ret != 0, S2N_ERR_CRL_INVALID_THIS_UPDATE);
```

Comment:
You don't have tests for this error, is that because it's not possible to create a crl that has invalid this_update/next_update data(not related to expiration/activation times)?

Reply:
Nah, I was just curious.